Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM bash:alpine3.16
+
+RUN apk add --no-cache vault
```

Comment:
Flipped over to using the `vault` image in the Dockerfile and adding bash to the container so we can take advantage of running the tests inside of bash instead of Alpine's default shell